install.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+PREFIX=/home/taste/tool-inst
+SOURCES=$(dirname $0)
+
+mkdir -p "${PREFIX}/include/TASTE-Linux-Drivers/src"
+cp -r "${SOURCES}/src/linux_ip_socket" "${PREFIX}/include/TASTE-Linux-Drivers/src"
```

Comment:
From TASTE-setup, I remember a rm to make sure that the destination does not contain any outdated stuff. Would this help here?

Reply:
Sure, this is good idea. Implemented & tested.